use-cases/in-memory-data-grid.html

```diff
@@ -58,60 +57,60 @@ <h1><strong>In-Memory Data Grid</strong> with Apache Ignite</h1>
-            sets originating from disjointed databases. Your applications can consume all the data from
-            Ignite as a single store while Ignite can keep the original databases in sync whenever in-memory
-            data gets updated.
-        </p>
-        <p>
-            However, there are some limitations if an external database is used as a persistence layer for
-            Ignite deployments. For instance, if you run Ignite SQL or scan queries, you need to ensure that
-            all the data has been preloaded to the in-memory cluster. Note that Ignite SQL or scan queries
-            can read data from disk only if it is stored in the native persistence.
-        </p>
-
-        <h2>Ignite Native Persistence</h2>
-        <p>Ignite native persistence is a distributed ACID and SQL-compliant disk store that transparently integrates
-            with Ignite in-memory layer. When the native persistence is enabled, Ignite stores both data and indexes on
-            disk and eliminates the time-consuming cache warm-up step. Since the native persistence always keeps a full
-            copy of data on disk, you are free to cache a subset of records in memory. If a required data record is
-            missing in memory, then Ignite reads it from the disk automatically regardless of the API you use -- be it
-            SQL, key-value, or scan queries.</p>
+        
+        <h2>What is an In-Memory Data Grid?</h2>
+
+        <p>An in-memory data grid is a read-through/write-through caching strategy in which the application 
+        layer treats the data grid as the primary data store. The application layer writes to and reads 
+        from the in-memory data grid. The in-memory data grid ensures that any underlying databases stay 
+        updated and consistent with the in-memory data.</p>
+
+        <p>The Ignite in-memory data grid provides all essential APIs needed to simplify its adoption. The 
+        APIs include <a href="/use-cases/key-value-store.html">distributed key-value</a> and <a href="/features/sql.html">ANSI SQL</a> queries, <a href="/features/acid-transactions.html">ACID transactions</a>, <a href="/features/collocated-processing.html">co-located processing</a>, 
+        and <a href="/features/machinelearning.html">machine learning</a> models. While key-value and SQL calls let you request, join, and group 
+        distributed data sets, the compute and machine learning components help to eliminate data 
+        shuffling over the network, thus, boosting compute and data-intensive calculations.</p>
+
+        <p>Ignite can store data both in memory and on disk with two options for data persistence. You 
+        can persist changes in an external database or have Ignite keep data in its <a href="/arch/native-persistence.html">native persistence</a>. </p>
+
+        <h2>IGNITE in-memory data grid AND EXTERNAL DATABASES</h2>
```

Comment:
We need to say "Ignite as an in-memory data grid ..." in headers to avoid any confusion that Ignite has a dedicated feature-component named similarly. "as an" highlights that it's one of the usage options. I'll update after merging changes.